grammar.js

```diff
@@ -480,8 +480,8 @@ module.exports = grammar({
 
     statements: $ => seq(
       $._statement,
-      repeat(seq($._semis, $._statement)),
-      optional($._semis),
+      repeat(seq($._semis_and_colon, $._statement)),
```

Comment:
I think we should keep _semis. I mean "semis" mean "semicolons", so semis_and_colon sounds
like we accept either semicolons or a colon.

The file was just using \r\n for _semis, but really it's because a newline is considered equivalent to a semicolon by the Kotlin compiler.

Reply:
i see, I'll try using [\r\n;] instead of [\r\n] for semis and test if parsing increases :D